ruby_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class TestHound
+  NAME = "Wormbots"
+  VERSION = '0.0.1'
+  LOGISTICS = [NAME, VERSION]
+
+  puts "Test"
+  puts "Test another"
+
+  def foo
+
+    puts "bar"
+
+    puts 'foo'
+    puts 'foo'
+
+    puts "bar"  
+    puts 'bar'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.